gtest_parallel.py

```diff
@@ -44,6 +44,11 @@
   import fcntl
 
 
+# The log file names is truncated to 256 to overcome OS limitations
```

Comment:
Can you move these constants into where they are used?

Reply:
Sure. Done